view/material/Form.kt

```diff
@@ -805,7 +805,7 @@ object Form {
     fun <T : Any> Dropdown(
         values: List<T>,
         selected: T?,
-        displayFn: (T) -> AnnotatedString = { AnnotatedString(it.toString()) },
+        displayFn: @Composable (T) -> AnnotatedString = { AnnotatedString(it.toString()) },
```

Comment:
We had an awkward situation where a method (`TypeDisplayName`) had 2 identical overloads, one Composable and one not. Making `displayFn` `@Composable` mitigates the need for the non-Composable overload.